From e99fc279858d939b833bdda85fd664ef4c335b09 Mon Sep 17 00:00:00 2001
From: Relintai <relintai@protonmail.com>
Date: Tue, 23 Feb 2021 10:14:50 +0100
Subject: [PATCH] Fix voxel editor after the skeleton editor patch.

---
 world/voxel_world_editor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/world/voxel_world_editor.h b/world/voxel_world_editor.h
index 7195940..9ae9717 100644
--- a/world/voxel_world_editor.h
+++ b/world/voxel_world_editor.h
@@ -91,7 +91,7 @@ protected:
 	void _notification(int p_what);
 
 public:
-	virtual bool forward_spatial_gui_input(Camera *p_camera, const Ref<InputEvent> &p_event) { return voxel_world_editor->forward_spatial_input_event(p_camera, p_event); }
+	virtual bool forward_spatial_gui_input(int p_index, Camera *p_camera, const Ref<InputEvent> &p_event) { return voxel_world_editor->forward_spatial_input_event(p_camera, p_event); }
 	virtual String get_name() const { return "VoxelWorldEditor"; }
 	bool has_main_screen() const { return false; }
 	virtual void edit(Object *p_object);
-- 
2.30.1

