diff -u a/tools/gen_sdk_package_darling_dmg.sh b/tools/gen_sdk_package_darling_dmg.sh
--- a/tools/gen_sdk_package_darling_dmg.sh	2018-12-20 16:06:04.744507679 +0000
+++ b/tools/gen_sdk_package_darling_dmg.sh	2018-12-20 16:06:39.000379232 +0000
@@ -33,22 +33,6 @@
 [ -n "$CC" ] && require $CC
 [ -n "$CXX" ] && require $CXX
 
-set +e
-
-command -v lsb_release 2>&1 > /dev/null
-
-if [[ $? -eq 0 ]] && [[ -n $(lsb_release -a 2>&1 | grep -i ubuntu) ]]; then
-  echo "Using ubuntu, skipping fuse module check"
-else
-  modinfo fuse &>/dev/null
-fi
-
-if [ $? -ne 0 ]; then
-  echo "Required kernel module 'fuse' not loaded" 1>&2
-  echo "Please run 'insmod fuse' as root" 1>&2
-  exit 1
-fi
-
 set -e
 
 pushd $BUILD_DIR &>/dev/null

